Runtime/ActionRouter/ActionRouterGroup.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using Padoru.Diagnostics;
+
+namespace Padoru.Core.ActionRouter
+{
+    public class ActionRouterGroup
+    {
+        public string Id { get; }
```

Comment:
This should be called ActionId or EventId or something that makes it clear that it corresponds to the ID of the action